tests/e2e/test_image.py

```diff
@@ -442,3 +442,54 @@ def test_external_image_build(
     if f"Successfully built {img_uri_str}" not in result.stdout:
         LOGGER.warning(result.stdout)
         raise AssertionError("Successfully built message was not found.")
+
+
+@pytest.mark.skipif(
+    sys.platform == "win32", reason="docker is not installed on Windows nodes"
+)
+@pytest.mark.skipif(
+    sys.platform == "darwin", reason="docker is not installed on Mac nodes"
+)
+def test_image_local_build(cli_runner: CLIRunner) -> None:
+    dockerfile_path = Path("nested/custom.Dockerfile")
+    dockerfile_path.parent.mkdir(parents=True)
+
+    random_file_to_disable_layer_caching = gen_random_file(dockerfile_path.parent)
+
+    with open(dockerfile_path, "w") as f:
+        f.write(
+            textwrap.dedent(
+                f"""\
+                    FROM python:3.9.7-alpine3.13
```

Comment:
We might reach the image pull request rate limit in the dockerhub (if we are super-actively developing and testing something), what if we use `ghcr.io/neuro-inc/alpine:latest`?